helm/db-operator/templates/_helpers.tpl

```diff
@@ -55,3 +55,38 @@ Create the name of the service account to use
 {{ default "default" .Values.serviceAccount.name }}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Common labels
+*/}}
+{{- define "db-operator.labels" -}}
+helm.sh/chart: {{ include "db-operator.chart" . }}
+{{ include "db-operator.selectorLabels" . }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- end -}}
+
+{{/*
+Selector labels
+*/}}
+{{- define "db-operator.selectorLabels" -}}
```

Comment:
when it's used?

Reply:
in line 64 ... but i can also combine both together ... was only copy paste this from the helm example how to name the currently.